public/app/plugins/panel/stat/types.ts

```diff
@@ -65,6 +72,33 @@ export function addStandardDataReduceOptions(
     showIf: currentConfig => currentConfig.reduceOptions.values === false,
   });
 
+  if (includeFieldMatcher) {
+    builder.addSelect({
+      path: 'reduceOptions.fields',
+      name: 'Fields',
+      description: 'Select the fields that should be included in the panel',
+      settings: {
+        options: [],
+        getOptions: (context: FieldOverrideContext) => {
+          const options = [
+            { value: '', label: 'Numeric Fields' },
+            { value: '/.*/', label: 'All Fields' },
+          ];
+          if (context && context.data) {
```

Comment:
@dprokop -- context and data are never set in panel tab, am I missing something?